pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -0,0 +1,183 @@
+package consensus
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/jonboulle/clockwork"
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/utils"
+)
+
+var info = capabilities.MustNewCapabilityInfo(
+	"ocr3",
+	capabilities.CapabilityTypeConsensus,
+	"OCR3 consensus exposed as a capability.",
+	"v1.0.0",
+)
+
+type capability struct {
+	services.StateMachine
+	capabilities.CapabilityInfo
+	store  *store
+	stopCh chan struct{}
```

Comment:
There is a `StopChan` type with built in methods:
```suggestion
	stopCh services.StopChan
```